lightspeed/modules/proc_create-lightspeed-instance.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: PROCEDURE
+
+[id="create-lightspeed-instance_{context}"]
+
+= Creating and deploying a {LightspeedShortName} instance
+
+Use this procedure to create and deploy a {LightspeedShortName} instance to your namespace.  
+
+.Prerequisites
+* You have created connection secrets for both {PlatformName} and {ibmwatsonxcodeassistant}.
```

Comment:
I realize this info will probably be in the previous section, but a link to the info about creating connection secrets might be helpful to the user here--up to you.